app/src/main/res/layout/view_unlock.xml

```diff
@@ -45,10 +44,85 @@
         android:gravity="center_horizontal"
         android:lines="2"
         android:maxLines="2"
-        android:text="@string/actions_enter_passcode_or_fingerprint" />
+        android:text="@string/actions_enter_passcode_or_fingerprint"
+        android:visibility="gone" />
 
     <com.saltedge.authenticator.widget.passcode.PasscodeInputView
         android:id="@+id/passcodeInputView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:layout_marginTop="@dimen/dp_115" />
+
+    <LinearLayout
+        android:id="@+id/resetPasscodeLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/backImageView"
+            android:layout_width="@dimen/dp_24"
+            android:layout_height="@dimen/dp_24"
+            android:layout_marginStart="@dimen/dp_16"
+            android:layout_marginTop="@dimen/dp_16"
+            android:scaleType="fitCenter"
+            android:src="@drawable/ic_back_24dp" />
+
+        <ImageView
+            android:id="@+id/imageView"
+            android:layout_width="@dimen/dp_328"
```

Comment:
try to not use absolute size in params that tends to match parent size. because not all screens have size that you propouse.
For example here, view width should be set as match_parent and it will have size = screen_size - margins.

Reply:
Ok, but I had extra margins(start/end)